main.tf

```diff
@@ -158,6 +158,7 @@ module "tls" {
 # Creates a public IP address and an Application Gateway for forwarding internet traffic to the GraphDB proxies/instances
```

Comment:
Have you tested deploying without the API GW and then switching `disable_agw` back to false?

Reply:
After latest commit it works now. The Vm's however need an upgrade once added to the backend pool